content/guide/setup.md

```diff
@@ -22,36 +21,51 @@ Now you’re ready to install and run Quire. And don’t worry, you only have to
 Open your your Terminal command line application and one-by-one copy and paste the following commands. First to install support software for Quire, and then Quire itself:
 
 
-1.) Install Apple’s Xcode if you don’t already have it:
+1.) Check if you have Apple's Xcode 9 installed by copying and pasting the following command:
+
+```
+which xcode-select
+```
+
+And install **Apple’s Xcode** if you don’t already have it:
 
 ```
 xcode-select --install
 ```
 
-2.) Install Homebrew, which helps manage other installs:
+2.) Install **[Homebrew](https://brew.sh)**, which helps manage other installs:
```

Comment:
To be consistent, should we link to Hugo, Node and others as well?

Reply:
I thought about that when I was working on the files but I wasn't sure because there are links to Hugo, Node, etc. also in about.md

Reply:
Good point, though people might only read a single page and not cover-to-cover. So maybe linking again is okay?

Reply:
I think it's ok. I'll keep this way of linking as a rule when things are mentioned in several pages